cli/tests/test_next_prev_commands.rs

```diff
@@ -322,3 +414,8 @@ fn test_next_editing() {
     Parent commit      : kkmpptxz 3fa8931e (empty) third
     "###);
 }
+
+fn get_log_output(test_env: &TestEnvironment, cwd: &Path) -> String {
+    let template = r#"separate(" ", change_id.short(), local_branches, description)"#;
```

Comment:
We use this template (or similar) in a bunch of other tests for what it's worth. Won't the default format print the email and timestamp, which are irrelevant clutter in these tests?

Reply:
If you can reduce it to get the parent>child lines back, I'm happy.

Reply:
Assuming you mean the line between the nodes below, they're only printed if there's branching or the message next to the node spans two lines. It looks like we can force them by printing two newlines in the template, but I don't really think that's necessary and it might add extra lines between nodes when there's branching.

```
◉  b
│
◉ a
```

Reply:
ok, leave it as is.